### DISCUSSION TITLE
How in the world is that suppose to work?

### DISCUSSION THREAD

**Discussion by frizouw** [Nicholaiii/midiscord#1] [Q&A] [answered]
Hello, 
I managed to find how to start your project in vs code I just don't know how to make it works, like how...how I am suppose to do stuff with that? I see token, is there a bot?

**Comment by lucaslaricchia** [comment]
Having the same problem here hehehe, it seems that the application is not even saving the token i tried

**Comment by Nicholaiii** [comment, marked as answer]
@frizouw @lucaslaricchia I appreciate the interest, the project is currently blocked because I'm unable to find an audio library that will allow me to mix audio sources, and the project is fairly pointless to me with only one simultaneous audio source.

**Reply by lucaslaricchia** [reply]
Oh its ok, first time i saw the project i thought it was to control the new feature "SoundBoard" on discord.
Thanks for the reply!
